jsonforms-tooling-common/src/index.ts

```diff
@@ -232,14 +236,15 @@ const asyncGenerateUiSchema = async (editorInstance: any, path: string) => {
  * @param {Object} schema the json schema, that will be validated
  * @param {function} callback forwards the current status to the caller
  */
-const validateJSONSchema = async (schema: Object) => {
+const validateUiSchema = async (schema: Object) => {
+  let valid = null;
   try {
     const ajv =  new Ajv();
-    ajv.compile(schema);
-    return true;
+    valid = ajv.validate(uiMetaSchema, schema);
```

Comment:
Return here directly, no need for the `valid` local var.

Reply:
Done